modules/manage/pages/iceberg/topic-iceberg-integration.adoc

```diff
@@ -435,6 +415,44 @@ FROM (
 
+== Performance considerations
+
+When you enable Iceberg for any substantial workload, you may see most of your cluster's CPU utilization increase as a result of translation of topic data to the Iceberg format. If this process overloads the brokers, Redpanda automatically applies backpressure to clients to prevent runaway lag of the Iceberg table behind the topic log.
+
+=== Use custom partitioning
+
+To improve query performance, consider implementing custom https://iceberg.apache.org/docs/nightly/partitioning/[partitioning^] for the Iceberg topic. Use the xref:reference:properties/topic-properties.adoc#redpanda-iceberg-partition-spec[`redpanda.iceberg.partition.spec`] topic property to define the partitioning scheme:
+
+[,bash,]
+----
+# Create new topic with five topic partitions, replication factor 1, and custom table partitioning for Iceberg
+rpk topic create <new-topic-name> -p5 -r1 -c "redpanda.iceberg.partition.spec=(<partition-key1>, <partition-key2>, ...)"
+----
+
+Valid `<partition-key>` values include a source column name or a transformation of a column. The columns referenced can be Redpanda-defined (such as `redpanda.timestamp`) or user-defined based on a schema that you register for the topic. The Iceberg table stores records that share the same partition key values in the same files based on this specification. 
+
+For example: 
+
+* To partition the table by a single key, such as a column `col1`, use: `redpanda.iceberg.partition.spec=(col1)`. 
+* To partition by multiple columns, use a comma-separated list: `redpanda.iceberg.partition.spec=(col1, col2)`. 
+* To partition by the year of a timestamp column `ts1`, and a string column `col1`, use: `redpanda.iceberg.partition.spec=(year(ts1), col1)`.
+
+For details on the partitioning specification, including allowed transforms, see the https://iceberg.apache.org/spec/#partitioning[Apache Iceberg documentation^].
+
+[TIP]
+====
+* It is recommended to partition by columns that you frequently use in queries. Columns with relatively few unique values, also known as low cardinality, are also good candidates for partitioning.
+* If you must partition based on columns with high cardinality, for example timestamps, use Iceberg's available transforms such as extracting the year, month, or day to avoid creating too many partitions. Too many partitions can offset the benefits of partitioning because more files need to be scanned and managed.
+====
+
+=== Avoid high column count
+
+A high column count results in more overhead when translating topics to the Iceberg table format. Redpanda recommends keeping the column count low for Iceberg topics to minimize the performance impact on your cluster.
+
+=== Size your cluster for Iceberg topics
+
+Iceberg workloads can be CPU-intensive and decrease throughput. To ensure that your cluster is sized appropriately for Iceberg topics, contact the Redpanda Customer Success team. 
```

Comment:
@tmgstevens Please confirm that this is all right to include re: reaching out to CS. Thanks!

Reply:
That's fine by me

Reply:
"Iceberg workloads can be CPU-intensive and can decrease throughput. In some cases, it may be necessary to  increase the size of your Redpanda cluster to accommodate the additional workload of Iceberg Topics. To ensure that your cluster is sized appropriately for Iceberg topics, contact the Redpanda Customer Success team. "